Neos.Flow/Tests/Unit/Mvc/View/JsonViewTest.php

```diff
@@ -152,7 +152,7 @@ public function jsonViewTestData()
      */
     public function testTransformValue($object, $configuration, $expected, $description)
     {
-        $jsonView = $this->getAccessibleMock(Mvc\View\JsonView::class, ['dummy'], [], '', false);
+        $jsonView = $this->getAccessibleMock(Mvc\View\JsonView::class, ['dummy'], [], '');
```

Comment:
Note: `false` prevents the constructor from being invoked, which means the default settings from `supportedOptions` are not applied. Therefore settings are empty and lead to accessing null.